app/controllers/outlets_controller.rb

```diff
@@ -71,10 +75,21 @@ def update
     end
     
     if @outlet.save
+      somebody_was_notified = ''
+      if action_performed != 'updated' && !@outlet.contact_emails.empty?
+        somebody_was_notified = "We have notified the following " + 
+        (@outlet.contact_emails.count == 1 ? "point" : "points") + 
```

Comment:
`pluralize` is designed for some of this logic, fwiw

https://api.rubyonrails.org/classes/ActiveSupport/Inflector.html